src/stories/tables.js

```diff
@@ -0,0 +1,27 @@
+import { storiesOf } from '@storybook/html';
```

Comment:
We could also have stories in each component folder 🤔

Reply:
with a `*.stories.js` extention, like suggested here:
https://storybook.js.org/basics/writing-stories/#loading-stories-dynamically